tern/analyze/common.py

```diff
@@ -452,3 +454,100 @@ def get_os_style(image_layer, binary):
                     package_manager=binary,
                     package_format=image_layer.pkg_format,
                     os_list=image_layer.os_guess), 'info'))
+
+
+def check_git_folder(path_to_folder):
+    '''Given a path to git folder, return the name and sha of repository.
+    This function looks into .git folder.'''
+    comment_line = ''
+    if os.path.isdir(path_to_folder):
+        # find .git folder
+        dot_git_folder_path = os.path.join(path_to_folder, '.git/')
+        if os.path.exists(dot_git_folder_path):
+            # find sha information
+            sha_info = ''
+            with open(dot_git_folder_path + 'HEAD', 'r') as head_file:
+                # .git/HEAD contains current head ref
+                ref_info = head_file.readline()
+                ref_relative_path = ref_info.split(' ', 1)[1].strip('\n')
+                sha_path = dot_git_folder_path + ref_relative_path
+                # get sha info from ref
+                with open(sha_path, 'r') as sha_file:
+                    sha_info = sha_file.readline().strip('\n')
+            # find project name
+            name_info = ''
+            conf = configparser.ConfigParser()
+            conf.read(dot_git_folder_path + 'config', encoding='utf-8')
+            # use remote url for git project name
+            if conf.has_section('remote "origin"'):
+                # url is the first item
+                project_url = conf.get('remote "origin"', 'url')
+                name_info = project_url.split('/')[-1]
+            else:
+                name_info = '(not found)'
+            comment_line = 'git project name: ' + name_info +\
+                           ' sha info: ' + sha_info
+        else:
+            comment_line = 'Not a git repository'
+    return comment_line
+
+
+def check_git_tar(path_to_tar):
```

Comment:
I don't think we need to look into the tarball that gets included into the docker image. It is already in there by the time we analyze it. I think this function can be deleted.